tools/timeit.c

```diff
@@ -74,57 +81,80 @@ static const char *g_target_redirect_stderr = 0;
 /* \brief If non-zero, append exit status at end of output file. */
 static int g_append_exitstats = 0;
 
+#ifdef _WIN32
+/* \brief If platform does not define rlim_t type, then define it as int. */
+typedef int rlim_t;
+
+/* \brief If platform does not define pid_t type, then define it as int. */
+typedef int pid_t;
+#endif
+
 /* @name Resource Limit Variables */
 /* @{ */
 
 /* \brief If non-sentinel, the CPU time limit to set for the target. */
-static rlim_t g_target_cpu_limit = ~(rlim_t) 0;
+static rlim_t g_target_cpu_limit = ~(rlim_t)0;
```

Comment:
These formatting changes would probably be good to leave out of the commit.

Reply:
fixed.